chaoscenter/graphql/server/pkg/projects/project_handler.go

```diff
@@ -2,75 +2,83 @@ package projects
 
 import (
 	"context"
+	"time"
+
+	"github.com/google/uuid"
 
-	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/graph/model"
 	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb"
 	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb/image_registry"
 	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb/project"
 	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/grpc"
-	image_registry2 "github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/image_registry"
 	"github.com/sirupsen/logrus"
```

Comment:
```suggestion
	log "github.com/sirupsen/logrus"
```
Can you use a `log` alias for maintenance?